Code/skyrim_ui/src/assets/i18n/ru.json

```diff
@@ -0,0 +1,156 @@
+{
+  "INFO": "Русская локализация Skyrim Together: Reborn! Версия: 1.3.0. Автор: <strong>Sidiusz</strong>",
+  
+  "COMPONENT": {
+    "CHAT": {
+      "SEND": "Отправить",
+      "MESSAGE": "Сообщение",
+      "MESSAGE_TOO_LONG": "Нельзя отправлять сообщение длиной более {{chatMessageLengthLimit}} символов."
+    },
+    "CONNECT": {
+      "INFO": {
+        "SERVER_ADDRESS": "Для подключения используйте полный формат адреса: <strong>айпи-адрес:порт</strong>. Стандартный порт: <strong>10578</strong>.",
+        "HELGEN": "НЕ подключайтесь ни к какому серверу, если вы не прошли вступительное задание \"На свободу\". Убедитесь, что вы покинули Хелген."
+      },
+      "ADDRESS": "Айпи-адрес",
+      "PASSWORD": "Пароль",
+      "SAVE_PASSWORD": "Сохранить пароль?",
+      "CONNECT": "Подключиться",
+      "CANCEL": "Отмена",
+      "PUBLIC_SERVERS": "Список серверов",
+      "ERROR": {
+        "CONNECTION": "Не удалось подключиться к серверу. Удостоверьтесь в правильности айпи-адреса и в корректной работе вашего Интернет-соединения",
+        "VERSION_MISMATCH": "Вы не можете подключиться из-за разницы в версии Skyrim Together между вами и сервером.",
+        "INVALID_ADDRESS": "Айпи-адрес имеет неверный формат."
+      }
+    },
+    "DISCONNECT": {
+      "PROMPT": "Вы уверены, что хотите отключиться?",
+      "PROCEED": "Подтвердить",
+      "CANCEL": "Отмена"
+    },
+    "ERROR": {
+      "OK": "Ок"
+    },
+    "NOTIFICATIONS": {
+      "MESSAGE_FROM": "Сообщение от "
+    },
+    "PARTY_MENU": {
+      "ACCEPT_INVITE": "Принять приглашение от {{name}}",
+      "LAUNCH_PARTY": "Создать группу",
+      "INVITES": "Приглашения",
+      "NO_INVITES": "Вас никто не пригласил.",
+      "EXPERIENCE_WARNING": "Для получения наилучшего опыта от прохождения заданий, следуйте нашим рекомендациям, которые вы можете найти по ссылке: <b>vk.com/skyrim_multiplayer</b>. Вот краткий список основных рекомендаций:",
```

Comment:
Our VK community has all the links to the project (website, github, wiki)
I replaced the link because the community is in Russian and Russian-speaking people will be available in a language they understand, and if they want details and the original text, all the links are there

Reply:
> 

Add both.

Reply:
We don't control those sources, and are thus unofficial. Point it to the official website.

Reply:
Added, sorry

Reply:
Adding both is fine too, but it still doesn't explicitly mention that the vk link is from an unofficial source. That needs to be mentioned.

Reply:
> Adding both is fine too, but it still doesn't explicitly mention that the vk link is from an unofficial source. That needs to be mentioned.

I do not think it is necessary because VK is a Russian social network (like facebook) and for the users of this network there are no safety risks

Reply:
No, it is necessary, since we, the maintainers, don't control that site. In fact, it's better if it's removed. Russian users can very easily use Google translate on our site, and find the vk board through other means. Resolve the merge conflicts, remove the vk link, and it can be merged.